torch/csrc/jit/passes/quantization.cpp

```diff
@@ -580,8 +568,20 @@ TORCH_API void InsertObservers(
     script::Module& module,
     const std::string& method_name,
     const QConfigDict& qconfig_dict) {
-  auto module_qconfig_map = getQConfigMap(module, qconfig_dict);
-  InsertObserversImpl(module, method_name, module_qconfig_map);
+  ModuleQConfigMap module_qconfig_map;
+  // Set of child modules that has the same qconfig
```

Comment:
This comment seems to be outdated - we no longer have childset which I believe it refers to.